algorithms/linfa-elasticnet/src/algorithm.rs

```diff
@@ -75,6 +152,28 @@ impl<F: Float, D: Data<Elem = F>> PredictInplace<ArrayBase<D, Ix2>, Array1<F>> f
     }
 }
 
+impl<F: Float, D: Data<Elem = F>> PredictInplace<ArrayBase<D, Ix2>, Array2<F>>
+    for MultiTaskElasticNet<F>
+{
+    /// Given an input matrix `X`, with shape `(n_samples, n_features)`,
+    /// `predict` returns the target variable according to elastic net
+    /// learned from the training data distribution.
+    fn predict_inplace(&self, x: &ArrayBase<D, Ix2>, y: &mut Array2<F>) {
+        assert_eq!(
+            x.nrows(),
+            y.nrows(),
+            "The number of data points must match the number of output targets."
+        );
```

Comment:
You also need to assert that `x.ncols() == y.ncols()` right?

Reply:
@YuhanLiin No. The number of columns in `x` corresponds to the number of features, while the number of columns in `y` is the number of tasks.

X_shape: (n_samples, n_features)
Y_shape: (n_samples, n_tasks)